pulseaudio-control.bash

```diff
@@ -320,6 +348,30 @@ function usage() {
     echo "    https://github.com/marioortizmanero/polybar-pulseaudio-control"
 }
 
+while [[ "$1" = --* ]]; do
+    case "$1" in
+        --vol-icon-low=*)
```

Comment:
These `--vol-icon-X` arguments imply that there are always going to be 3 icons for the volume. What if there was just a `--vol-icon X` that appended the icon to `VOLUME_ICONS`? Or a single `--vol-icon X,Y,Z` that split the icons at the commas. This script is meant to work for any font, and some have 2 or 4 icons for the volume.

Reply:
Ah sure, ok.